package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.0.0",
   "scripts": {
     "ng": "ng",
-    "start": "ng serve",
+    "start": "ng serve --configuration=en",
```

Comment:
The configuration flag can be removed, since the default configuration is set in angular.json.

Reply:
I have made the suggested changes.